### ISSUE TITLE
The default value for the ObfuscatorAttribute is different

### ISSUE THREAD

**Issue by mituha** [lextm/obfuscar#55]
https://msdn.microsoft.com/en-us/library/system.reflection.obfuscationattribute.exclude(v=vs.110).aspx

**Comment by lextm** [comment]
Can you be more specific on what is the problem?

**Comment by mituha** [comment]
```
[Obfuscation]
public class DefaultExcludeClass
{
	public void Test()
	{
		var attr = this.GetType().GetCustomAttribute<ObfuscationAttribute>();
		Console.WriteLine("Exclude = {0} : {1}", attr.Exclude, this.GetType().Name);
	}
}
```
Exclude = True : a　　

Exclude property is true, but it has been obfuscated.

There is no problem if explicitly specify.
```
[Obfuscation(Exclude = false)]
public class ExcludeFalseClass
{
	public void Test()
	{
		var attr = this.GetType().GetCustomAttribute<ObfuscationAttribute>();
		Console.WriteLine("Exclude = {0} : {1}", attr.Exclude, this.GetType().Name);
	}
}
```
Exclude = False : A

```
[Obfuscation(Exclude = true)]
public class ExcludeTrueClass
{
	public void Test()
	{
		var attr = this.GetType().GetCustomAttribute<ObfuscationAttribute>();
		Console.WriteLine("Exclude = {0} : {1}", attr.Exclude, this.GetType().Name);
	}
}
```
Exclude = True : ExcludeTrueClass

**Comment by mituha** [comment]
TypeDefinitionExtensions.cs
```
var rename = !(bool)(Helper.GetAttributePropertyByName (attr, "Exclude") ?? false);
```
Why GetAttributePropertyByName returns a null?

**Comment by lextm** [comment]
Fixed in 6a337f39043c77edf97b8f31da133281dd44131f.
